openbb_platform/platform/provider/openbb_provider/standard_models/etf_countries.py

```diff
@@ -26,7 +26,6 @@ def upper_symbol(cls, v: Union[str, List[str], Set[str]]):
 class EtfCountriesData(Data):
     """ETF Countries Data."""
 
-    country: str = Field(description="The country of the exposure.")
-    weight: float = Field(
-        description="Exposure of the ETF to the country in normalized percentage points."
+    country: str = Field(
+        description="The country of the exposure.  Corresponding values are normalized percent points."
```

Comment:
@piiq, if this is a reasonable place for describing the scale, I will apply the same treatment to Sectors and Info as a separate PR.